rotkehlchen/chain/ethereum/modules/liquity/decoder.py

```diff
@@ -60,10 +61,20 @@ def __init__(
         self.eth = A_ETH.resolve_to_crypto_asset()
         self.lqty = A_LQTY.resolve_to_evm_token()
 
-    def _decode_trove_operations(self, context: DecoderContext) -> DecodingOutput:
+    def _decode_trove_operations(
+            self,
+            context: DecoderContext,
+            post_decoding: bool = False,
+    ) -> DecodingOutput:
         if context.tx_log.topics[0] != BALANCE_UPDATE:
             return DEFAULT_DECODING_OUTPUT
 
+        if self.base.get_proxy_owner(context.transaction.to_address) is not None and post_decoding is False:  # type: ignore[arg-type]  #  `transaction.to_address` is not None here  # noqa: E501
+            # If this is a transaction made via a DS Proxy, it needs to be handled in a
+            # post-decoding rule. Returning `matched_counterparty`` only here and not for other
```

Comment:
Don't really see why you need to add the backticks in all of your comments. This is not a docstring and it's not rst or markdown so it won't appear anywhere rendered in any special way

```suggestion
            # post-decoding rule. Returning `matched_counterparty` only here and not for other
```